src/Components/ProductListingPage/ProductsSection/ProductsCard/ProductsCard.jsx

```diff
@@ -1,16 +1,13 @@
-import { useCart } from "../../../../Context/CartContext";
-import { useWishlist } from "../../../../Context/WishlistContext";
+import { useCart, useWishlist } from "../../../../Context/index";
```

Comment:
you don't need to specify an index, it '..../Context' in the end works the same way

Reply:
I didn't know this. Thank you for pointing it out.